vnext/Shared/RuntimeOptions.h

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <string>
+
+namespace Microsoft::React {
+
+/// <summary>
+/// Sets a global boolean value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global key</param>
+void __cdecl SetRuntimeOptionBool(std::string &&name, bool value) noexcept;
+
+/// <summary>
+/// Sets a global signed integer value identified by an arbitrary string.
+/// </summary>
+/// <param name="name">Global boolean key</param>
+void __cdecl SetRuntimeOptionInt(std::string &&name, std::int32_t value) noexcept;
+
+/// <summary>
+/// Retrieves a global boolean value for the given key.
+/// </summary>
+/// <param name="name">Global boolean key</param>
+/// <returns>Value stored for the given key, or false if the entry doesn't exist (default)</returns>
+const bool __cdecl GetRuntimeOptionBool(const std::string &name) noexcept;
```

Comment:
>const bool [](start = 0, length = 10)

For my education: What's the benefit of declaring a value return type 'const'?

Reply:
Can't remember the details, but @NikoAri suggested this.
I think it was related to the value guaranteed to not be modified in the invoked function allowing for a compile-time optimization.

Reply:
std::string_view would be nice!

Returning "const bool" shouldn't be needed, "bool" should be good!

---
In reply to: [470318324](https://github.com/microsoft/react-native-windows/pull/5713#discussion_r470318324) [](ancestors = 470318324)